core/src/avm2/globals/flash/events/TouchEvent.as

```diff
@@ -36,23 +36,13 @@ public class TouchEvent extends Event {
     public var ctrlKey: Boolean; // On Windows or Linux, indicates whether the Ctrl key is active (true) or inactive (false).
     public var altKey: Boolean; // Indicates whether the Alt key is active (true) or inactive (false).
     public var shiftKey: Boolean; // Indicates whether the Shift key is active (true) or inactive (false).
-    public var commandKey: Boolean; // Indicates whether the command key is activated (Mac only).
-    public var controlKey: Boolean; // Indicates whether the Control key is activated on Mac and whether the Ctrl key is activated on Windows or Linux.
-    public var timestamp: Number; // Reports the time of the event in relative milliseconds.
-    public var touchIntent: String; // Reports whether the touch was generated by the primary or the eraser end of a stylus.
-    public var isTouchPointCanceled: Boolean; // Reports that this touch input sequence was canceled by the operating system.
-    public var isRelatedObjectInaccessible: Boolean; // If true, the relatedObject property is set to null for reasons related to security sandboxes.
-    private var _stageX: Number; // [read-only] The horizontal coordinate at which the event occurred in global Stage coordinates.
```

Comment:
Should `_stageX` and `_stageY` be removed? `stageX` and `stageY` are properties of TouchEvent in normal AS (non-AIR).

Reply:
> Should _stageX and _stageY be removed? stageX and stageY are properties of TouchEvent in normal AS (non-AIR).

I did not mean to remove isRelatedObjectInacessible and _stageX & StageY, they should not be removed as they are part of the flash API, i'll fix that